code/modules/unit_tests/designs.dm

```diff
@@ -0,0 +1,38 @@
+/datum/unit_test/designs
+
+/datum/unit_test/designs/Run()
+	var/datum/design/defaultDesign = new /datum/design()
+	var/datum/design/nanites/defaultDesignNanites = new /datum/design/nanites()
+	var/datum/design/surgery/defaultDesignSurgery = new /datum/design/surgery()
```

Comment:
Use snake_case variable names and use `allocate`, it'll qdel the datum when the test finishes.

Reply:
Use snake_case variables everywhere else BTW.

Reply:
Changed everything to camel_case and allocate doesn't work as discussed (added comment at the top of run()